<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>393</width>
    <height>193</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>400</height>
   </size>
  </property>
  <property name="focusPolicy">
   <enum>Qt::WheelFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>1</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <property name="verticalSpacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>30</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QTabWidget" name="test">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="update">
      <attribute name="title">
       <string>update</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="spacing">
        <number>6</number>
       </property>
       <item row="0" column="1" colspan="3">
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>40</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QCheckBox" name="gujian">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>12</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>固件</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="ziku">
          <property name="geometry">
           <rect>
            <x>63</x>
            <y>12</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="tabletTracking">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>字库</string>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="qita">
          <property name="geometry">
           <rect>
            <x>116</x>
            <y>12</y>
            <width>47</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>其他</string>
          </property>
         </widget>
        </widget>
       </item>
       <item row="2" column="0" colspan="4">
        <widget class="QProgressBar" name="progressBar">
         <property name="styleSheet">
          <string notr="true">QProgressBar{
border: 1px solid grey;
border-radius: 5px;
text-align: center;
}
QProgressBar::chunk{
background-color: #CD96CD;
width: 10px;
margin: 0.5px;
}</string>
         </property>
         <property name="value">
          <number>0</number>
         </property>
         <property name="textVisible">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="open">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>open</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QLineEdit" name="edit_open">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::DefaultContextMenu</enum>
         </property>
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color:transparent;</string>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
         <property name="clearButtonEnabled">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>208</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="send">
         <property name="text">
          <string>send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>nomal</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="topMargin">
        <number>6</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <property name="bottomMargin">
        <number>6</number>
       </property>
       <property name="horizontalSpacing">
        <number>3</number>
       </property>
       <item row="0" column="0" colspan="4">
        <widget class="QTextEdit" name="textEdit"/>
       </item>
       <item row="1" column="0">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>181</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="1">
        <widget class="QCheckBox" name="isHex">
         <property name="text">
          <string>hex</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QCheckBox" name="addCR">
         <property name="text">
          <string>CR&lt;\n&gt;</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
